src/prisma/schema.prisma

```diff
@@ -33,6 +33,7 @@ model AI {
   listInPublicCatalog Boolean      @default(false) @map("list_in_public_catalog")
   options             Json?
   profile             Json?
+  categoryName        CategoryType       @default(AI_MODELS)
```

Comment:
This shouldn't be needed right, since we are switching to a one-to-many relationship?